components/form/test/form.test.js

```diff
@@ -5,12 +5,29 @@ import './nested-form.js';
 import { defineCE, expect, fixture } from '@brightspace-ui/testing';
 import { html, LitElement } from 'lit';
 
+class TestTwoForms extends LitElement {
+	render() {
+		return html`
+			<d2l-form id="nested-form-1">
+				<input type="text" aria-label="Input 1" name="input1" required>
+			</d2l-form>
+			<d2l-form id="nested-form-2">
+				<input type="text" aria-label="Input 2" name="input2" required>
+			</d2l-form>
+		`;
+	}
+}
```

Comment:
I had to pull this up here instead of using `defineCE` because you can't embed things that were also created using `defineCE` in a thing using `defineCE` -- at least I couldn't get it to work.